### DISCUSSION TITLE
【数字进制转换工具】

### DISCUSSION THREAD

**Discussion by suwia** [he3-app/he3-feedback#111] [Bug Report]
### Version
base-conversion 1.0.0

### Environment
Mac v1.3.2

### What is expected?
希望对【数字进制转换工具】添加二进制的小数转换功能

**Comment by lyzhang1999** [comment]
[感谢您的建议，我们已建立任务跟踪这个需求。](https://github.com/he3-app/he3-feedback/issues/112)

**Reply by lyzhang1999** [reply]
另外能否给一个参考工具链接呢？

**Reply by suwia** [reply]
这个我看了下issue，和 issue #103 需求差不多，就是浮点数和二进制的转换工具，最好有正负数的。
